deps/3rdparty/cassowary-0.60/cassowary-0.60-Windows.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>Project [](start = 1, length = 7)

Looks like most cassowary libs were updated but one in x86 was just deleted. why?